dts/arm/gigadevice/gd32f4xx/gd32f4xx.dtsi

```diff
@@ -579,6 +579,14 @@
 				#pwm-cells = <3>;
 			};
 		};
+
+		mac: ethernet@40028000 {
+			compatible = "gd,gd32-ethernet";
+			reg = <0x40028000 0x8000>;
+			label = "ETH_0";
```

Comment:
remove `label` property.

Reply:
Can you explain the reason? Many of devices above have labels, e.g. TIMER_13.